client/search-ui/src/components/LastSyncedIcon.tsx

```diff
@@ -14,15 +14,34 @@ interface Props {
 }
 
 export const LastSyncedIcon: React.FunctionComponent<React.PropsWithChildren<Props>> = props => {
-    const formattedTime = format(Date.parse(props.lastSyncedTime), 'yyyy-MM-dd pp')
+    const parsedDate = new Date(Date.parse(props.lastSyncedTime))
+    const formattedTime = format(parsedDate, 'yyyy-MM-dd pp')
+    const oneDayAgo = new Date()
+    const oneWeekAgo = new Date()
+    oneDayAgo.setDate(oneDayAgo.getDate() - 1)
+    oneWeekAgo.setDate(oneWeekAgo.getDate() - 7)
+
+    let color = 'currentColor'
+    let status = `Last synced: ${formattedTime}`
+    let icon = mdiCloudCheckOutline
+    if (parsedDate < oneWeekAgo) {
+        color = 'var(--danger)'
+        status = 'Warning: severely out of date, last synced: ' + formattedTime + '. Please contact your administrator.'
+        icon = mdiCloudAlert
+    } else if (parsedDate < oneDayAgo) {
+        color = 'var(--warning)'
+        status = 'Warning: slightly out of date, last synced: ' + formattedTime
```

Comment:
string template here as well:

```suggestion
        status = `Warning: slightly out of date, last synced: ${formattedTime}`
```